### DISCUSSION TITLE
automatic setting of the system language at installationa

### DISCUSSION THREAD

**Discussion by Nael-Sayegh** [BearWare/TeamTalk5#1307] [Ideas]
Is it possible that when you install TeamTalk on Windows, the software is installed with the system language of the computer?

**Comment by bear101** [comment]
I'm not familiar with a way for [Qt](https://www.qt.io/) to enable a language based on system language. It would be quite complicated to implement, e.g. in Canada they speak both French and English. In Brazil they speak Portuguese...

**Comment by Nael-Sayegh** [comment]
Thank you for your reply.
What I was trying to explain is that the software could get the language 
used by the computer and automatically switch to it.

Lien de mon site et de ma chaîne youtube sous ma signature
Cordialement
Nael Sayegh
https://www.nael-accessvision.com/
Chaîne youtube: nael accessvision
https://www.youtube.com/c/naelaccessvision

Le 01/05/2022 à 12:48, Bjørn Damstedt Rasmussen a écrit :

**Comment by CoBC** [comment]
An alternative can be to ask the user to choose his language at first startup. But I don't know how to include a combobox in QMessageBox.

**Comment by DalisonCPU** [comment]
You can take the language of the Teamtalk 5 installation program with Inno setup, and pass the command line to TeamTalk 5 at the end of the installation, removing the user option to choose whether to run it or not, opening it automatically for the first time, and then scanning the existing files to find the selected teamtalk5 language
